bot/controllers/verify.js

```diff
@@ -5,6 +5,8 @@ const models = require('../../db/models');
 const uuidv4 = require('uuid/v4');
 const nodemailer = require('nodemailer');
 const { generateCode } = require('../botUtils.js');
+const { validDomains } = require('../botUtils');
```

Comment:
Since botUtils is already required on line 7 you could include validDomains on that same line. Something like `const { generateCode, validDomains } = require('../botUtils.js');`